src/AuthModule/Components/VerifyAccount/VerifyAccount.tsx

```diff
@@ -1,7 +1,165 @@
+  
+}
+
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const onSubmit = async (data:any) => {
+    setIsLoading(true)
+    handleData(data)
+    // handleRegister(values);
+  };
+
+ 
   return (
-    <div>verifyaccount</div>
-  )
+    <>
+      <div className="auth-container vh-100  d-flex flex-column justify-content-center align-items-center">
+        <div className="logo">
+          <img className="form-logo pb-2 " src={logo} alt="logo" />
+        </div>
+        <form
+          onSubmit={handleSubmit(onSubmit)}
+          className="form  col-xl-5 col-lg-6 col-md-7 col-sm-9 col-10 bg-info rounded-4 p-sm-5 p-3"
+        >
+         <p className="text-light mb-0 mt-3">welcome to PMS</p>
+          <h2 className="form-Name ">
+            <span>V</span>erify Account
+          </h2>
+
+          <div className="  d-flex flex-column gap-3  mt-4">
+            {/* <div className=" one-input-group">
+              <div
+                className="form-outline my-2  text-start   "
+                data-mdb-input-init
+              >
+                <label
+                  className="form-label  fw-medium mb-0"
+                >
+                  E-mail
+                </label>
+                <input
+                  {...register("email", {
+                    required: "Email is required",
+                    pattern: {
+                      value: /[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}/,
+                      message: "Please enter a valid email address",
+                    },
+                    maxLength: {
+                      value: 30,
+                      message: "Email must be at most 30 characters",
+                    },
+                    minLength: {
+                      value: 10,
+                      message: "Email must be at least 10 characters",
+                    },
+                  })}
+                  className="form-input form-control bg-transparent rounded-bottom-0  border-0 border border-bottom text-white p-1"
+                  type="text"
+                  placeholder="Enter your E-mail"
+                  aria-label="readonly input example"
+                  autoComplete="off"
+                />
+                {errors?.email && (
+                  <p className="mt-1  text-danger">{errors?.email?.message}</p>
+                )}
+              </div>
+            </div> */}
+
+            <div className=" one-input-group">
+              <div className="form-outline position-relative text-start d-flex flex-wrap">
+                <label
+                  className="w-100 form-label fw-medium mb-0"
+                >
+                  OTP Verification
+                </label>
+                <input
+                  {...register("code", {
+                    required: {
+                      value: true,
+                      message: "OTP is Required",
+                    },
+                      maxLength: {
```

Comment:
Consider adding only the validations required from the backend, the OTP is usually sent as 4 characters, but we should assume that it should be at least 4 characters or at most 8 characters.  As this is not stated from the backend. 
If the backend changes its mind and makes the code 10 characters long, users will not be able to register.